sections/register-form.liquid

```diff
@@ -192,6 +213,11 @@
         "type": "text",
         "label": "Terms and agreements",
         "default": "Terms and agreements"
+      }, {
+        "id": "terms_and_agreements_button_label",
```

Comment:
Can we rename this to something like `terms_and_agreements_accept_button_label`?